Project Euler/Problem 4/problem4_sol1.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
```

Comment:
Could we have the description of the problem in the header so that we don't need to visit the Project Euler website? You could also include a reference link here to facilitate this future visit.

Reply:
Thanks. I've copied the description from [sol1.rb](https://github.com/TheAlgorithms/Ruby/blob/master/project_euler/problem_4/sol1.rb).

Should be a link to https://projecteuler.net/problem=4 added?

Reply:
I think if we have the description, the link becomes useless. Thanks for the changes.